dojo/utils.py

```diff
@@ -2124,3 +2124,8 @@ def get_enabled_notifications_list():
         if get_system_setting('enable_{}_notifications'.format(choice[0])):
             enabled.append(choice[0])
     return enabled
+
+
+def is_finding_feature_group_enabled():
```

Comment:
```suggestion
def is_finding_groups_enabled():
```
The `feature_` part is not necessary here (would be in the wrong place anyway). And `groups` with an `s` in the end lets it fit better to the model

Reply:
Done